### DISCUSSION TITLE
Restrict Position true and minZoom

### DISCUSSION THREAD

**Discussion by kschmott** [ValentinH/react-easy-crop#448] [Q&A]
Hey all,

I was just curious if there was a way to make it impossible to zoom out past the image? 
![image](https://user-images.githubusercontent.com/103714838/221384074-15c3497d-4881-480d-931d-0c89c2fe441f.png)
Here is an example of what I am talking about. I have tried messing with the minZoom but I can't seem to devise an algorithm that will properly limit the zoom whilst also allowing for the maximum zoom out ability. I am essentially trying to copy instagram cropping if that helps.

 Thanks!

**Comment by ValentinH** [comment]
I think this could be done by using the original ratio (width/height or the other way around) and have the parent container aspect be 1(square) like on Instagram. 
If you cannot have the parent being a square, then you also have to take the parent aspect ratio in parameter when computing the min zoom.

**Reply by kschmott** [reply]
Thanks for the comment. I am able to have the "parent container" be a square but I am still not sure would I would calculate the minZoom based off of that?

**Reply by ValentinH** [reply]
For example, if your image is 800x600px, the minZoom could be set to 600/800 => 0.75. I didn't try it but this is what I would try first.
